### ISSUE TITLE
Clarify settings, specially user/pass

### ISSUE THREAD

**Issue by fbarcelo2** [tprasadtp/protonvpn-docker#182]
### Version of `protonvpn-docker`

NA

### Details about Feature/Enhancement

I'm new with this container, I've readed the warnings saying that `PROTONVPN_USERNAME` and  `PROTONVPN_PASSWORD`   shoud be filled with OpenVPN Username and OpenVPN Password besides of being counterintuitive I understand that at some point I should setup Open VPN and give my accounts details to it. 
But in host machine? in the container itself? How? Are there any special parameters that should be given?
So basically where should I put my PROTONVPN_USERNAME` and  `PROTONVPN_PASSWORD and how should I setup the OpenVpn? May anyone clarify this?

### Code of Conduct & PII Redaction

- [X] I agree to follow this project's Code of Conduct
- [X] I have removed any sensitive personally identifying information(PII) and secrets from in this issue report.

**Comment by Ludofloria** [comment]
Hello there !

Those should be set in the environment variables of your docker container.

If you use docker-compose.yml, you could do :

```yml
  environment:
     # Credentials
     PROTONVPN_USERNAME: "yourOpenVPNusername"
     PROTONVPN_PASSWORD: "yourOpenVPNpassword"
     # Override these where applicable
     PROTONVPN_SERVER: "VPNCountry"
     PROTONVPN_EXCLUDE_CIDRS: "192.168.1.0/24,172.16.0.0/24"
```

For better security, those can be set in the `.env` variable, right next to your `docker-composer.yml` file.

```yml
# docker-compose.yml
  environment:
    # Credentials
    PROTONVPN_USERNAME: ${PROTONVPN_USERNAME}
    PROTONVPN_PASSWORD: ${PROTONVPN_PASSWORD}
    # Override these where applicable
    PROTONVPN_SERVER: ${PROTONVPN_SERVER:-NL}
    PROTONVPN_EXCLUDE_CIDRS: ${LOCAL_CIDR}
```

```
# .env
PROTONVPN_USERNAME=yourOpenVPNusername
PROTONVPN_PASSWORD=yourOpenVPNpassword
PROTONVPN_SERVER=NL
LOCAL_CIDR=192.168.1.0/24,172.168.0.0/16
```

If you're not using `docker-compose`, but the regular `docker run`, you just have to add the args `-e PROTONVPN_USERNAME=yourOpenVPNUsername -e PROTONVPN_PASSWORD=yourOpenVPNpassword`
